packages/sdk/src/expression.test.ts

```diff
@@ -222,24 +222,38 @@ describe("lint expression", () => {
   });
 
   test("allow safe string methods", () => {
-    expect(
-      lintExpression({
-        expression: `title.toLowerCase()`,
-        availableVariables: new Set(["title"]),
-      })
-    ).toEqual([]);
-    expect(
-      lintExpression({
-        expression: `title.replace(" ", "-")`,
-        availableVariables: new Set(["title"]),
-      })
-    ).toEqual([]);
-    expect(
-      lintExpression({
-        expression: `title.split("-")`,
-        availableVariables: new Set(["title"]),
-      })
-    ).toEqual([]);
+    const stringMethods = [
+      "toLowerCase",
+      "replace",
+      "split",
+      "at",
+      "endsWith",
+      "includes",
+      "startsWith",
+      "toUpperCase",
+      "toLocaleLowerCase",
+      "toLocaleUpperCase",
+    ];
+    for (const method of stringMethods) {
```

Comment:
try test.each instead so each method would have separate case in test report.
https://vitest.dev/api/#test-each